governance/polls/Launch Maker Teleport with 1 basis point fee - May 23, 2022.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Launch Maker Teleport with 1 basis point fee - May 23, 2022
+summary: Signal your support or opposition to Launch Maker Teleport with 1 basis point fee.
+discussion_link: https://forum.makerdao.com/t/signal-request-launch-maker-teleport-with-1-basis-point-fee/15109
+vote_type: Plurality Voting
+categories:
+   - Bridge
+   - Rates
+   - Medium Impact
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2022-05-23T16:00:00
+end_date: 2022-05-26T16:00:00
+---
+# Poll: Launch Maker Teleport with 1 basis point fee - May 23, 2022
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the Maker Community. This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning on Monday, May 23 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the following changes:
+* Launch Maker Teleport with 1 basis point fee (0,01%)
+
+Please review the discussion [thread](https://forum.makerdao.com/t/signal-request-launch-maker-teleport-with-1-basis-point-fee/15109) to help inform your position before voting.
+
+## Outcomes
+
+**If the votes for the 'Yes' option exceed the votes for the 'No' option then the following actions will be taken:**
+* This change will be included in an upcoming Executive Vote as the Protocol Engineering Core Unit's schedule allows.
+* It is expected that this Executive Vote will take place within 30 days of this poll passing, absent external factors.
+* If the Friday executive passes, then this change will become active in the Maker Protocol after the [GSM Pause Delay](https://community-development.makerdao.com/en/learn/governance/param-gsm-pause-delay) has expired.
```

Comment:
Please change the GSM link to: https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay
Please also change "If the Friday executive passes" to "If the executive passes". Execs aren't on Fridays anymore. The template is outdated